src/graphene-private.h

```diff
@@ -25,6 +25,7 @@
 #define __GRAPHENE_PRIVATE_H__
 
 #include "config.h"
+#include <stdio.h>
```

Comment:
Some files are using fprintf() and stderr(), namely graphene-box.c and graphene-vectors.c.  I just moved the stdio.h inclusion into these files directly, if that is better.